### ISSUE TITLE
iOS - Search - Search page briefly goes blank when switching workspace filter

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#53359]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.69-1
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** Y
**Email or phone of affected tester (no customers):** applausetester+232803@applause.expensifail.com
**Issue reported by:** Applause Internal Team

## Action Performed:
1. Launch ND or hybrid app.
2. Go to Search.
3. Open workspace switcher.
4. Select another workspace (create a workspace first if there isn't any).
5. Open workspace switcher.
6. Tap Clear filter.

## Expected Result:
Search page will not go blank when switching workspace filter.

## Actual Result:
Search page briefly goes blank when switching workspace filter.

## Workaround:
Unknown

## Platforms:

- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [x] iOS: HybridApp
- [x] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/2e3f06f6-1398-4c37-9398-4526504ec8be
  
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by mountiny** [comment]
@chrispader @hannojg @kirillzyusko moat likely related to the native stack change

Not blocking the user so demoted

**Comment by kirillzyusko** [comment]
Hello, I'm Kiryl from Margelo expert agency and I would like to work on this issue 👋

**Comment by mjasikowski** [comment]
@kirillzyusko any updates here?

**Comment by kirillzyusko** [comment]
@mjasikowski yes, I've been investigating this, and for me it looks like the problem is somewhere in native code of `react-native-screens`. I fully removed layout of the Search screen and left only red rectangle that should be always shown, but the bug still exist:

https://github.com/user-attachments/assets/876fafc6-49f0-447f-b664-357dd1e28e70

I spoke with @WoLewicki but he is not aware about this issue. I also asked @hannojg to look into this problem as well. We should have a new update tomorrow/the day after tomorrow 🤞

**Comment by hannojg** [comment]
@chrispader will fix this issue while working on a fix for this one:

- https://github.com/Expensify/App/issues/53533#issuecomment-2536162308

**Comment by zanyrenney** [comment]
I have sent a DM to @fedirjh to complete the checklist

**Comment by garrettmknight** [comment]
$250 approved for @fedirjh
